ydb/core/kqp/federated_query/kqp_federated_query_actors.h

```diff
@@ -62,9 +78,20 @@ class TDescribeSchemaSecretsService: public NActors::TActorBootstrapped<TDescrib
         size_t FilledSecretsCnt = 0;
     };
 
+    struct TRequestContext {
+        THolder<TEvResolveSecret> Request;
+        TRetryPolicy::IRetryState::TPtr RetryState;
+
+        TRequestContext(THolder<TEvResolveSecret> request)
```

Comment:
Тут тоже можно добавить 

```suggestion
        explicit TRequestContext(THolder<TEvResolveSecret> request)
```